serverless-aws-typescript/package.json

```diff
@@ -0,0 +1,13 @@
+{
+		"name": "${PROJECT}",
+		"devDependencies": {
+				"@types/node": "^14"
+		},
+		"dependencies": {
+				"@pulumi/aws": "5.10.0",
+				"@pulumi/aws-apigateway": "^0.0.11",
+				"@pulumi/awsx": "^1.0.0-beta",
+				"@pulumi/pulumi": "^3.0.0",
+				"typescript": "^4.0.0"
+		}
```

Comment:
When I run this with `pulumi up`, I get:

```
Diagnostics:
  pulumi:providers:aws (default_4_30_0):
    error: no resource plugin 'pulumi-resource-aws' found in the workspace at version v4.30.0 or on your $PATH, install the plugin using `pulumi plugin install resource aws v4.30.0`
```

`serverless-aws-python` and `-go` show the same, though -- so I'm guessing that means this is expected for now?

Reply:
Hmm - I have no idea why it would be creating a 4.30 AWS provider for you.  The packages here all use 5.x.

Let me reverify from a clean environment.

Reply:
I can't repro this one either, and can't figure out what would cause you to see this unless you had a binary on path that is overriding the declared dependencies here. Going to resolve, but if you still see this - let me know.